src/main/java/com/sourcegraph/cody/MyAccountTabPanel.kt

```diff
@@ -79,7 +81,16 @@ class MyAccountTabPanel(val project: Project) : JPanel() {
             button("Upgrade") { BrowserUtil.browse(ConfigUtil.DOTCOM_URL + "cody/subscription") }
         upgradeButton.component.putClientProperty(DarculaButtonUI.DEFAULT_STYLE_KEY, true)
       }
-      button("Check Usage") { BrowserUtil.browse(ConfigUtil.DOTCOM_URL + "cody/manage") }
+      button("Manage Account") {
+        val username =
+            CodyAuthenticationManager.getInstance(project)
+                .getActiveAccount()
+                ?.let(CodyAccount::name)
+        BrowserUtil.browse(
+            ConfigUtil.DOTCOM_URL +
+                "cody/manage?cody_client_user=" +
+                URLEncoder.encode(username, "UTF-8"))
```

Comment:
This will throw a NullPointerException if `username` is null, but I guess that's impossible given we know this is a free account, it must have a name...

Reply:
I just updated it to explicit handle null accounts explicitly